src/models/fetch_org_response.rs

```diff
@@ -15,7 +15,7 @@ use serde_json::Value;
 #[derive(Debug, Clone, PartialEq, Default, Serialize, Deserialize)]
 pub struct OrgMetadata {
     #[serde(flatten)]
-    metadata: HashMap<String, Value>,
+    pub metadata: HashMap<String, Value>,
```

Comment:
this one's a no-brainer

Reply:
Yes, let me pull it out from this PR, I am not blocked on that.